ImageItem3D/ImageItem3D.js

```diff
@@ -7,10 +7,10 @@ import PropTypes from 'prop-types';
 import {useCallback, useImperativeHandle, useRef, useState} from 'react';
 import * as THREE from 'three';
 
-import Skinnable from '../Skinnable';
+import {get3DShape} from '../internal/util/util';
```

Comment:
You can shorten this to: import {get3DShape} from '../internal/util';
I would also recommend to name this "utils", it is more common.